### PR TITLE
Add tooltip to Custom Insert button

### PR DESCRIPTION
**Pull request by reubenjsinclair**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: src/modules/PublicLab.CustomInsert.js [hunk 1]
```diff
@@ -67,7 +67,13 @@ module.exports = function CustomInsert(_module, wysiwyg) {
     }
   });
   const builder = require("./PublicLab.CustomInsert.Template.js");
-  $('.wk-commands').append('<a class="woofmark-command-insert btn btn-outline-secondary" data-toggle="Insert" title="Custom Insert"><i class="fa fa-tags"></i></a>');
+
```
Comment (NARUDESIGNS) [inline src/modules/PublicLab.CustomInsert.js:70]: Good job, just a little change here, line 70 shouldn't be a blank line. That's where the code additions should start.

Comment (reubenjsinclair) [reply src/modules/PublicLab.CustomInsert.js]: Thanks! Is the new commit better?

Comment (NARUDESIGNS) [reply src/modules/PublicLab.CustomInsert.js]: Looks good to me! 
Well done 🥳

File: src/modules/PublicLab.CustomInsert.js [hunk 2]
```diff
@@ -67,7 +67,12 @@ module.exports = function CustomInsert(_module, wysiwyg) {
     }
   });
   const builder = require("./PublicLab.CustomInsert.Template.js");
-  $('.wk-commands').append('<a class="woofmark-command-insert btn btn-outline-secondary" data-toggle="Insert" title="Custom Insert"><i class="fa fa-tags"></i></a>');
+  $('.wk-commands').append('<button class="woofmark-command-insert btn btn-outline-secondary" data-toggle="insert" title="Custom Insert"><i class="fa fa-tags"></i></button>');
```
Comment (TildaDares) [inline src/modules/PublicLab.CustomInsert.js:70]:
```suggestion
  $('.wk-commands').append('<button class="woofmark-command-insert btn btn-outline-secondary" data-toggle="Insert" title="Custom Insert"><i class="fa fa-tags"></i></button>');
```

Can you just change the `i` in `Insert` to uppercase? Thanks @reubenjsinclair!

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Review by TildaDares — APPROVED** [review]
Great job @reubenjsinclair! Thank you

**Comment by jywarren** [comment]
Just needed to sync this with an upstream test fix and I think it should be good now!

**Comment by jywarren** [comment]
Great! Merged and many thanks!!! 🎉

**Comment by NARUDESIGNS** [comment]
> Just needed to sync this with an upstream test fix and I think it should be good now!

I need to find similar fix for the [wysiwyg PR](https://github.com/publiclab/PublicLab.Editor/pull/785)
I remember you mentioned rebasing. But please I'd appreciate it if you could reiterate please. 
Do I need to rebase this to main? or what exactly can I do?
I tried pulling changes and pushing again but it still doesn't pass the ui-test here on Github.

Also, I noticed my other PRs passes the ui-tests on Github and are fine. Just this one is affected.

@jywarren

**Pull request merged**